src/expression-modifiers/normalization/index.js

```diff
@@ -0,0 +1,114 @@
+import util from '../../utils/util';
+import helper from '../helper';
+import propertyPanelDef from './properties';
+
+const DEFAULT_OPTIONS = {
+  type: 'normalization',
+  disabled: false,
+  primaryDimension: 0,
+  relativeNumbers: 0,
+  crossAllDimensions: false,
+  showExcludedValues: true,
+  fullNormalization: false,
+  outputExpression: '',
+};
+
+const maxNumDimensionsSupported = 2;
+
+function getAggrOneDim(expression, dim1) {
+  return `Aggr(${expression}, ${dim1})`;
+}
+
+function getAggrDisregadSelec(expression, dim1) {
+  return `Aggr({1}${expression}, ${dim1})`;
+}
+
+function getTotal(expression) {
+  return `total ${expression}`;
+}
+function getSum(expression) {
+  return `Sum (${expression})`;
+}
+
+function getDivide(measureExp, expression) {
+  return `${measureExp}/${expression}`;
+}
+
+function getSumDisregardSelec(expression) {
+  return `Sum({1} ${expression})`;
+}
+
+export default {
+  translationKey: 'properties.modifier.normalization',
+
+  needDimension: helper.needDimension,
+
+  isApplicable({ properties, layout }) {
+    return helper.isApplicable({
+      properties,
+      layout,
+      minDimensions: 1,
+      maxDimensions: maxNumDimensionsSupported,
```

Comment:
To make it more consistent, you could remove the `maxNumDimensionsSupported` variable, since it's only used here, and there are no constant for min dimensions...